x-pack/plugin/eql/qa/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/eql/EqlRestValidationIT.java

```diff
@@ -17,21 +17,19 @@
 
     @Override
     protected String getInexistentIndexErrorMessage() {
-        return "\"caused_by\":{\"type\":\"verification_exception\",\"reason\":\"Found 1 problem\\nline -1:-1: Unknown index ";
+        return "\"root_cause\":[{\"type\":\"index_not_found_exception\",\"reason\":\"no such index ";
```

Comment:
"Qualifying" the index in the exception replaces the VerificationException with an IndexNotFoundException, to ["harmonise"](https://github.com/elastic/elasticsearch/pull/76076/files#diff-88b74c40b99fe71512274733a696fcbf1cb53c34362b2b75892498d4d232cc0eR236) the failure reporting. This would be needed to not differentiate the failure format depending on which request parameters are set.